src/components/composer/composer/composer.ts

```diff
@@ -86,6 +86,13 @@ css/* scss */ `
       }
     }
 
+    .o-composer[contenteditable="true"]:empty:not(:focus)::before {
```

Comment:
I think [contenteditable="true"] is useless
```suggestion
    .o-composer:empty:not(:focus)::before {
```